### ISSUE TITLE
Getproperty method doesn't work

### ISSUE THREAD

**Issue by sachinlobo11** [ysfchn/DynamicComponents-AI2#69]
I have used this extension for reading metaproperty details. 
I was getting error got argument 0 expected 4.

I have checked the code I don't know what is a problem .

I wanted to read the text data from label ( getter).
I have also tried get property with name "Text". Couldn't read anything. Is there anything to do.. 
Can someone help?

**Comment by ysfchn** [comment]
Can you show your blocks? Are you using the latest version of the extension?

**Comment by sachinlobo11** [comment]
No problem. Get property worked for me.  GetPropertyMeta seems to have problem. Thats fine because I deleted those block and did alternative using set property.
![blocks (2)](https://github.com/user-attachments/assets/f3f76bc6-b9a6-455a-a29a-4aec76abf887)

My Actual goal is to when I click on dynamic component(label) it(Text) should underline using  `<u>name </u>` But  `name` variable could only contains only few charecters. (sometime 20 charecters sometimes 4 charecters).

So I have planed to extract from meta property. meta property block looks like this.
![blocks (1)](https://github.com/user-attachments/assets/9564fdea-87cf-4db6-9228-9f1536b91e22)

So here it was saying while running in my Phone "Provided 0 argument need 4". Currently running on android 14.

**Comment by ysfchn** [comment]
Yeah, it seems that `GetPropertyMeta` blocks having problems. But you are supposed to use `any component` blocks (recommended) or `GetProperty` block anyway if you want to get a property value of a component. `GetPropertyMeta` returns ALL properties' information as a dictionary, not just values.

![resim](https://github.com/user-attachments/assets/9c2e961d-6d68-4a5e-a6cf-fa00d1b0b2ab)

Also, since you already have a `get component` variable, you don't need to `GetId` of it and get its component back again. You can just use it in `set Label Text` block.
